src/bin/edit/localization.rs

```diff
@@ -924,17 +923,17 @@ pub fn init() {
 
     for l in langs {
         lang = match l.as_str() {
-            "en" => LangId::en,
-            "de" => LangId::de,
-            "es" => LangId::es,
-            "fr" => LangId::fr,
-            "it" => LangId::it,
-            "ja" => LangId::ja,
-            "ko" => LangId::ko,
-            "pt-br" => LangId::pt_br,
-            "ru" => LangId::ru,
-            "zh-hant" => LangId::zh_hant,
-            "zh" => LangId::zh_hans,
+            l if l.starts_with("en") => LangId::en,
+            l if l.starts_with("de") => LangId::de,
+            l if l.starts_with("es") => LangId::es,
+            l if l.starts_with("fr") => LangId::fr,
+            l if l.starts_with("it") => LangId::it,
+            l if l.starts_with("ja") => LangId::ja,
+            l if l.starts_with("ko") => LangId::ko,
+            l if l.starts_with("pt-br") => LangId::pt_br,
+            l if l.starts_with("ru") => LangId::ru,
+            l if l.starts_with("zh-hant") => LangId::zh_hant,
+            l if l.starts_with("zh-hans") => LangId::zh_hans,
```

Comment:
There's another issue with this code: It should ideally perform ASCII case-insensitive prefix matching (similar to [eq_ignore_ascii_case](https://doc.rust-lang.org/std/primitive.str.html#method.eq_ignore_ascii_case) but as `starts_with`). If you're up for it, it'd be great if you could add such a method to `helpers.rs` and use it here.

Otherwise, if you're short on time, we can of course also just merge this PR and do this some time later.

Reply:
I'll work on this tonight :-) Thanks for the suggestions!